### PR TITLE
Update chat template so no users-icon shown when a server or private conversation is selected.

### LINKED ISSUE #622: Remove User List icon when viewing Server Connection
1. Click the Server Connection on the left pane
2. Look in the right hand upper corner

Observed:
There is a User List icon, that does nothing when you click it.

Expected:
That shouldn't be there. Better would be the settings (gear) icon for that server.

### PR DESCRIPTION
**Pull request by MaxLeiter**
Fixes #622, should I add a font-awesome settings icon even if it won't do anything at the moment?

### CODE REVIEW

**Comment by xPaw** [comment]
:+1: 

> should I add a font-awesome settings icon even if it won't do anything at the moment?

Nah. Channels eventually should have settings too.

**Comment by YaManicKill** [comment]
Definitely don't add a settings icon yet.

However, yeah, I don't think you should set this up as if the settings is only going to be on the lobby. I would just show user on anything other than lobby or user.

**Comment by MaxLeiter** [comment]
Thats how it works with his PR @YaManicKill

**Comment by astorije** [comment]
> should I add a font-awesome settings icon even if it won't do anything at the moment?

Definitely not, or we'll have a lot of "Settings don't work" issues popping up :-)

This is an off-topic side note that we should keep for later (the user settings design doc I was mentioning in #604), but I'm concerned about UI/UX regarding having a Settings button at the bottom left of the app (for global settings) and on the top right of each channel (for channel settings). I have ideas about that and will share them.

File: client/views/chat.tpl [hunk 1]
```diff
@@ -2,7 +2,11 @@
 <div id="chan-{{id}}" data-title="{{name}}" data-id="{{id}}" data-type="{{type}}" class="chan {{type}}">
 	<div class="header">
 		<button class="lt"></button>
-		<button class="rt"></button>
+		{{#equal type "lobby"}}
+			{{!-- Insert settings icon --}}
+		{{else}}
+			<button class="rt"></button>
+		{{/equal}}
```
Comment (astorije) [inline client/views/chat.tpl]: I'm very not fond of placeholders like this one for potential future code/UI elements that might not be located there. Could you instead check that the type is _not_ lobby then display the button, without `else` statement please?

**Comment by astorije** [comment]
By the way, thanks a lot for that!! :-)

File: client/views/chat.tpl (hunk 1 above)
Comment (astorije) [inline client/views/chat.tpl]: Actually, checking that the type is a channel is probably more appropriate to make sure it doesn't show up on lobbies and private messages. Anything else we should display or not display this?

Comment (xPaw) [inline client/views/chat.tpl]: You're right, changing it to `{{#equal type "channel"}}` will do the job.

**Comment by astorije** [comment]
@MaxLeiter, just to confirm, is `shout.templates.js` up-to-date with your change on `chat.tpl`?

**Comment by MaxLeiter** [comment]
I forgot to run it a second time; will update tonight and message in IRC when I do.

**Comment by MaxLeiter** [comment]
@astorije looks like I did update the template; ready to pull :)

**Comment by astorije** [comment]
:+1:

**Comment by MaxLeiter** [comment]
Worth noting that checking if its a channel also solved the issue of a user icon showing in private conversations.

**Pull request closed without merging**